### PR TITLE
Add Rari transfer flow

### PR DESCRIPTION
**Pull request by graszka22**

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -1467,7 +1469,9 @@
       "youAreWithdrawingFromYieldPool": "You are withdrawing from Yield pool",
       "youAreWithdrawingFromEthPool": "You are withdrawing from ETH pool",
       "withdrawDetails": "Withdraw details",
-      "rariWithdrawalFee": "Rari withdrawal fee"
+      "rariWithdrawalFee": "Rari withdrawal fee",
+      "youAreTransfering": "You are transfering",
```
Comment (IAmKio) [inline src/locales/en/common.json:1473]:
```suggestion
      "youAreTransferring": "You are transferring",
```

File: src/screens/Rari/RariInfo.js [hunk 2]
```diff
@@ -101,17 +97,13 @@ const RariLogo = styled(CachedImage)`
...
   rates,
+  rariFundBalance,
 }: Props) => {
-  useEffect(() => {
-    fetchRariFundBalance();
-  }, []);
-
   const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+  const totalRariFundBalance = (Object.values(rariFundBalance): any).reduce((sum, balance) => sum + balance, 0);
+  const totalRariFundBalanceInFiat = convertUSDToFiat(totalRariFundBalance, rates, fiatCurrency);
```
Comment (IAmKio) [inline src/screens/Rari/RariInfo.js:106]: Just curious to know what happens if this fails? Do we have a fallback value or logic?

File: src/screens/Rari/RariTransfer.js [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+    [amount, selectedContact, useGasToken],
+  );
+
+  useEffect(() => {
+    updateTxFeeDebounced();
+    return updateTxFeeDebounced.cancel;
+  }, [updateTxFeeDebounced]);
+
+  const handleFormSubmit = async () => {
+    if (submitPressed) return; // double press
```
Comment (IAmKio) [inline src/screens/Rari/RariTransfer.js:150]: Very nice 👍

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a minor point but otherwise LGTM!

File: src/screens/Rari/RariTransfer.js [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+  isEstimating,
+  estimateErrorMessage,
+  userDepositInRariToken,
+  contacts,
+  addContact,
+  estimateTransaction,
+  useGasToken,
+  balances,
+  resetEstimateTransaction,
+}) => {
```
Comment (wjkg) [inline src/screens/Rari/RariTransfer.js:107]: I'd only like to add that it looks like prop types are missing here: 
```suggestion
}: Props) => {
```

File: src/screens/Rari/RariInfo.js (hunk 2 above)
Comment (graszka22) [reply src/screens/Rari/RariInfo.js:106]: `rariFundBalance` is kept in redux store and persisted, so if fetching fund balance fails (or any other rari data, like user's balances, prices etc.), 0 will be showed or older value that was fetched recently.
Additionally every time it fails a toast is displayed like `Cannot fetch Rari data. Please try again` or graph error box, if graph call failed.

**Pull request merged**